auto_gptq/modeling/_base.py

```diff
@@ -1299,6 +1345,25 @@ def skip(*args, **kwargs):
             )
             model.load_state_dict(checkpoint)
 
+        kerenl_backend_type = dynamically_import_QuantLinear(
+            use_triton=use_triton,
+            desc_act=quantize_config.desc_act,
+            group_size=quantize_config.group_size,
+            bits=quantize_config.bits,
+            disable_exllama=disable_exllama,
+            disable_exllamav2=disable_exllamav2,
+            use_qigen=use_qigen,
+            disable_marlin=not use_marlin,
+        )
+
+        if not quantize_config.new_checkpoint_format:
+            model = convert_new_checkpoint_format(
+                model,
+                True,
+                quantize_config,
+                kerenl_backend_type
+            )
```

Comment:
Can you also add the argument `is_legacy_format: Optional[bool] = None` to `from_quantized` that allows to specify it in case not in the `config`, defaulting first to the quantization config value if any, and else to the user specified if any, and else to `True`?

For example, if somebody is quantizing models with an external library, the quantization config may not contain a `is_legacy_format`.